README.md

```diff
@@ -16,10 +16,20 @@
 ## Description
 This terraform module creates an S3 Bucket with the options of adding other s3 bucket configurations.
 
-Examples available [here](https://github.com/boldlink/terraform-aws-s3/tree/main/examples)
+### Reasons to Use this Module over Stand Alone Resources
+- This module offers simplicity through detailed examples, making it user-friendly.
+- It follows AWS security best practices by utilizing checkov to ensure compliance.
+- It simplifies the process of setting up s3 bucket with the desired configurations
+- The module provides support for lifecycle configuration.
+- In response to a recent AWS S3 update, this module now includes ownership control, allowing users to specify ownership types. The default is `ObjectWriter`
+- Encryption is enabled by default, allowing users to utilize existing AWS Customer Master Keys (CMKs) or create new ones using the module. Additionally, it supports Server-Side Encryption (SSE).
+- It includes support for S3 bucket notifications, enabling users to set up event-driven workflows.
+- Versioning is enabled by default, providing automatic version control.
```

Comment:
I would think versioning should be disabled by default, this increases the cost on s3 and not all files are to backup, what do you think?

Reply:
Yeah, I think this makes sense

Reply:
I have now disabled it by default, updated README and added checkov exception for the same